vpn/options.go

```diff
@@ -140,7 +152,9 @@ func (o *Options) HasAuthInfo() bool {
 
 const clientOptions = "V1,dev-type tun,link-mtu 1549,tun-mtu 1500,proto %sv4,cipher %s,auth %s,keysize %s,key-method 2,tls-client"
 
-func (o *Options) String() string {
+// string produces a comma-separated representation of the options, in the same
+// order and format that theopenvpn server expects from us.
+func (o *Options) string() string {
```

Comment:
You want to keep this function public to implement [the Stringer interface](https://go.dev/tour/methods/17)